### DISCUSSION TITLE
Possible Mistaken Entry? Quasar PGC 2831485

### DISCUSSION THREAD

**Discussion by brent1123** [Stellarium/stellarium#2016] [Q&A]
I noticed this object appearing near double star HR8680 south of Fomalhaut and took a short series of exposures on it. I was surprised to see a Quasar listed as Mag +6 but had never shot one before and didn't have much else in mind to shoot that night.

However, after processing the photo I was plate solving and adding labels to the image noting various stars and background objects, and found no references to this object in any database. I searched on SIMBAD, centered on the coordinates of HR8680 (as they are nearly on top of each other) and even with the radius set to a few arcminutes, no objects matching the description were found.

I posted about this on Cloudy Nights as well, where you can also see screenshots of the object in Stellarium as well as my capture. Is this just a mistaken entry? 

https://www.cloudynights.com/topic/797326-how-to-verify-if-i-captured-quasar-pgc-2831485/#entry11481640

**Comment by gzotti** [comment]
I cannot reproduce this. I see no quasar <mag14 in PsA. I can only assume a temporary glitch in automated processing and download of HyperLEDA data. Force a QSO data update to see if it's still here.
